xcdat/variable.py

```diff
@@ -0,0 +1,103 @@
+"""Variable module for Dataset data variable related functions."""
+import xarray as xr
+
+
+def open_variable(dataset: xr.Dataset, var_name: str) -> xr.DataArray:
+    """Opens a data variable of type DataArray with additional attributes.
+
+    This function calls DataArray accessors on the data variable to store
+    additional attributes, which are cached in the DataArray object.
+
+    Attributes include:
+
+    - Axis bounds information ("lat_bnds", "lon_bnds", "time_bnds")
+
+    Parameters
+    ----------
+    dataset : xr.Dataset
+        The parent Dataset.
+    var_name : str
+        The name of the variable to be opened.
+
+    Returns
+    -------
+    xr.DataArray
+        The variable with additional attributes.
+
+    Examples
+    --------
+    Import module:
+
+    >>> from xcdat.dataset import open_dataset
+    >>> from xcdat.variable import open_variable
+
+    Open a variable from a Dataset:
+
+    >>> ds = open_dataset("file_path") # Autogenerates bounds if missing
+    >>> tas = open_variable(ds, "tas")
+
+    Get lat bounds attribute:
+
+    >>> tas.bounds.lat
+
+    Get lon bounds attribute:
+
+    >>> tas.bounds.lon
+
+    Get time bounds attribute:
```

Comment:
Usage examples for `open_variable()`. Viewable in docs [here](Example usage for `open_dataset()`. Viewable in the docs [here](https://xcdat.readthedocs.io/en/feature-80-open-dataset/generated/xcdat.dataset.html).